lambda/tile_ingest_lambda.py

```diff
@@ -87,8 +105,10 @@ def handler(event, context):
     tile_index_result = tile_index_db.getCuboid(msg_data["chunk_key"], int(msg_data["ingest_job"]))
     if tile_index_result is None:
         # If chunk_key is gone, another lambda uploaded the cuboids and deleted the chunk_key afterwards.
-        # Remove message so it's not redelivered.
-        ingest_queue.deleteMessage(msg_id, msg_rx_handle)
+        if not sqs_triggered:
```

Comment:
Why is it only deleting message when not sqs_triggered, does the message automatically get pulled from the queue and added to an async lambda call?

Reply:
When triggering via SQS, AWS handles message deletion for you.